packages/design-system/src/components/Review/Review.tsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import ReviewLink from './ReviewLink';
+import classNames from 'classnames';
+
+export type ReviewHeadingLevel = '1' | '2' | '3' | '4' | '5';
+
+export interface ReviewProps {
+  /**
+   * `Review` component's body content.
+   */
+  children: React.ReactNode;
+  className?: string;
+  /**
+   * Optional label to give screenreaders longer, more descriptive text to
+   * explain the context of an edit link.
+   */
+  editAriaLabel?: string;
+  /**
+   * An optional node in place of the edit link. If this defined, no edit link will be shown.
+   */
+  editContent?: React.ReactNode;
+  /**
+   * Href for the edit link. If this is undefined, no edit link will be shown.
+   */
+  editHref?: string;
+  editText?: React.ReactNode;
+  heading?: React.ReactNode;
+  /**
+   * Heading type to override default `<h3>`.
+   */
+  headingLevel?: ReviewHeadingLevel;
+  /**
+   * An optional function that is executed on edit link click. The event and
+   * props.editHref value are passed to this function.
+   */
+  onEditClick?: (...args: any[]) => any;
```

Comment:
General question that I also asked in my Tabs PR -- should we be more strictly typing these callback functions? Like with set parameters?

Reply:
I don't know that we should unless the function is defined with a particular set of params that should be typed.. in this case it's typed generically as a func so the caller can do anything they want with onEditClick.

Reply:
If we opt to more strictly type these props, it should probably happen after the initial TS conversion (with a major release). I'm personally OK with keeping the type interfaces as they are and revising later since I imagine other interfaces need reviewed as well.

Reply:
yep agreed, could run into troubles with integration